packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -133,19 +133,28 @@ function getDefaultHandlers(modules: ValidatorFnsModules, options: GossipHandler
     const blockInputMeta =
       config.getForkSeq(signedBlock.message.slot) >= ForkSeq.deneb ? blockInputRes.blockInputMeta : {};
 
-    metrics?.gossipBlock.receivedToGossipValidate.observe(recvToVal);
```

Comment:
this previously was incorrect because it was observing the latency not the actual rec to validation time, fixed below